cfme/infrastructure/config_management.py

```diff
@@ -4,6 +4,7 @@
 
 
 import cfme
+import cfme.base.credentials
```

Comment:
**REQUIRED:** As this is a new line and in ALL the other files we have imported the object and not the module, can we do this here too perhaps using an ```as``` to distinguish?